decksite/templates/rotation.mustache

```diff
@@ -4,14 +4,12 @@
     {{#runs}}
         <p>{{runs}} ({{runs_percent}}%) of 168 runs completed. {{num_cards}} cards have hits.</p>
         <p>Stats on undecided cards are redacted to reduce rampant market manipulation.</p>
-        {{#rotation_query}}
-            <p>
-                <form class="inline">
-                    <input type="text" name="rq" value="{{rotation_query}}">
-                    <button type="submit">Search</button>
-                </form>
-            </p>
-        {{/rotation_query}}
+        {{#in_rotation}}
+            <button id="toggle-filters-button" onclick="PD.filter.toggleDisplayFilter()">Show filters</button>
```

Comment:
If this is identical each time it shows up let’s make it a partial.  You can include partials

{{> likethis}}